integration/addr/manager.go

```diff
@@ -22,3 +49,21 @@ func Suggest() (port int, resolvedHost string, err error) {
 	resolvedHost = addr.IP.String()
 	return
 }
+
+// Suggest suggests an address a process can listen on. It returns
+// a tuple consisting of a free port and the hostname resolved to its IP.
+// It makes sure that new port allocated does not conflict with old ports
+// allocated within 1 minute.
+func Suggest() (port int, resolvedHost string, err error) {
+	for i := 0; i < 100; i++ {
```

Comment:
nit: Can we maybe also pull up that random number into a `const`

(If we do that, we also want to change the err message in line 67 and the doc in 53-56)

Reply:
Done